homeassistant/components/camera/__init__.py

```diff
@@ -748,6 +763,26 @@ async def async_get_webrtc_client_configuration(self) -> WebRTCClientConfigurati
 
         return config
 
+    @final
+    @callback
+    def get_camera_capabilities(self) -> CameraCapabilities:
```

Comment:
Should we add the `@cache` decorator since we only need to calculate this once?

Reply:
We need to recalculate it if the WebRTC provider changes, but we will look into it tomorrow.

Reply:
Nick told me we should use `@cache` only on a class or static functions. Otherwise, `self` will be added to the cache and, therefore, never garbage collected. Also, we can only invalidate the whole cache, not only for one instance.

I converted the function to a property so I can use `@under_cached_property`